skills/morph-ppt/reference/morph-helpers.py

```
@@ -0,0 +1,392 @@
+
+            # Method 1: name-based unghosted detection
+            unghosted = _check_unghosted(curr_data, prev_slide)
+            if unghosted:
+                print(f"{YELLOW}  Warning: Found unghosted content from slide {prev_slide}:{NC}")
+                for item in unghosted:
+                    print(f"     {item}")
+                print(f"{YELLOW}     These shapes should be ghosted to x=36cm{NC}")
+                has_error = True
+            else:
+                print(f"{GREEN}  No unghosted content detected{NC}")
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)